python/tank/platform/engine.py

```diff
@@ -1447,7 +1507,11 @@ def _create_dialog(self, title, bundle, widget, parent):
         :type widget: :class:`PySide.QtGui.QWidget`
         """
         from .qt import tankqdialog
-        
+
+        # TankQDialog uses the bundled core font. Make sure they are loaded
+        # since know we have a QApplication at this point.
+        self._ensure_core_fonts_loaded()
```

Comment:
We are currently running this in three different places in the engine. do you think there would be a way to call it just once?